packages/fuels-test-helpers/src/node.rs

```diff
@@ -230,32 +233,45 @@ fn write_temp_config_file(config: Value) -> NamedTempFile {
     config_file.unwrap()
 }
 
-pub fn spawn_fuel_service(
+pub async fn new_fuel_node(
     coins: Vec<(UtxoId, Coin)>,
     consensus_parameters_config: Option<ConsensusParameters>,
-    free_port: Port,
+    socket_addr: SocketAddr,
 ) {
+    let (tx, rx) = oneshot::channel();
```

Comment:
I think I understand why we're using tokio's `onyshot` channels here; we want to wait for a signal from the `tokio::spawn` below that it has finished setting up the node. Is that right?

If so, could you write a tiny comment line explaining this rationale? It might confuse people who are new to Rust and channel-based async stuff.

Reply:
Yes, you're right. I'll do it.